bin/live/pycbc_live_single_trigger_fits

```diff
@@ -136,25 +148,36 @@ elif args.duration_bin_spacing == 'linear':
 
 logging.info("Finding files")
 
-files = [f for f in os.listdir(os.path.join(args.top_directory, args.analysis_date))
-         if args.file_identifier in f]
+
+files = find_trigger_files(
+            args.directory,
+            args.gps_start_time,
+            args.gps_end_time,
+            id_string=args.file_identifier,
+            date_directories=args.date_directories,
+            date_directory_format=args.date_directory_format
+        )
 
 logging.info("{} files found".format(len(files)))
 
+logging.info("Setting up the cut dictionaries")
 # Add template duration cuts according to the bin inputs
 args.template_cuts = args.template_cuts or []
 args.template_cuts.append(f"template_duration:{min(duration_bin_edges)}:lower")
 args.template_cuts.append(f"template_duration:{max(duration_bin_edges)}:upper_inc")
 
+# Ensure that triggers are within the time limits
+args.trigger_cuts = args.trigger_cuts or []
+args.trigger_cuts.append(f"end_time:{args.gps_start_time}:lower_inc")
```

Comment:
This cut will help in the case the the gps_start_time (and equivalently gps_end_time) are in the middle of a file